docs/core/porting/modernize.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Modernize your upgraded .NET Framework apps
+description: This article demonstrates some of the ways you can modernize your .NET Framework app after you've upgraded to .NET
+author: adegeo
+ms.date: 05/25/2023
+dev_langs: ["csharp", "vb"]
+no-loc: ["package.config", PackageReference]
+---
+
+# Modernize after upgrading to .NET from .NET Framework
+
+In this article, you'll learn about different ways you can modernize your .NET Framework app after it's been upgraded to .NET. Use the [.NET Upgrade Assistant](upgrade-assistant-overview.md) tool to upgrade your app to .NET.
+
+## Missing APIs
+
+When upgrading a .NET Framework app, you'll most likely have some incompatibilities. This is because .NET Framework is a Windows-only technology and .NET is a cross-platform technology. Some libraries aren't For example, .NET doesn't provide out-of-the-box APIs to access the Windows Registry like .NET Framework did. Support for the Windows Registry is provided by the `Microsoft.Win32.Registry` NuGet package. Many the .NET Framework-specific libraries have been ported to .NET or .NET Standard, and are hosted on NuGet. If you find a missing reference in your project, search NuGet.
```

Comment:
nit (missing period):

```suggestion
When upgrading a .NET Framework app, you'll most likely have some incompatibilities. This is because .NET Framework is a Windows-only technology and .NET is a cross-platform technology. Some libraries aren't. For example, .NET doesn't provide out-of-the-box APIs to access the Windows Registry like .NET Framework did. Support for the Windows Registry is provided by the `Microsoft.Win32.Registry` NuGet package. Many the .NET Framework-specific libraries have been ported to .NET or .NET Standard, and are hosted on NuGet. If you find a missing reference in your project, search NuGet.
```